src/js/angular/core/directives/queryeditor/templates/query-editor.html

```diff
@@ -103,25 +103,25 @@
                         <button class="btn btn-link" id="wb-sparql-copyToClipboardQuery"
                                 ng-hide="notoolbarCopy"
                                 ng-click="copyToClipboardQuery()"
-                                tooltip="Get URL to current query"
+                                tooltip="{{'core.popover.get.url.current.query' | translate}}"
                                 tooltip-append-to-body="true">
                             <span class="icon-link icon-2-5x"></span>
                         </button>
                         <button class="btn btn-link" id="inference"
                                 ng-hide="notoolbarInference"
                                 ng-click="currentQuery.inference = !currentQuery.inference"
-                                tooltip="Include inferred data in results: {{currentQuery.inference ? 'ON' : 'OFF'}}"
+                                tooltip="{{createTooltipMsg('core.include.inferred', currentQuery.inference)}}"
                                 tooltip-append-to-body="true">
                             <span class="icon-2-5x"
                                   ng-class="{'icon-inferred-on':currentQuery.inference, 'icon-inferred-off':!currentQuery.inference}"></span>
                         </button>
                         <button class="btn btn-link" id="sameAs"
                                 ng-hide="notoolbarSameAs"
                                 ng-click="currentQuery.sameAs = !currentQuery.sameAs"
-                                tooltip="Expand results over owl:sameAs: {{currentQuery.sameAs ? 'ON' : 'OFF'}}"
+                                tooltip="{{createTooltipMsg('core.expand.sameAs', currentQuery.sameAs)}}"
                                 tooltip-append-to-body="true">
                             <span class="icon-2-5x"
-                                  tooltip="{{!currentQuery.inference ? 'Requires \'Include Inferred\'!' : ''}}"
+                                  tooltip="{{!currentQuery.inference ? 'core.require.inferred' : '' | translate}}"
```

Comment:
Is it not meant to be `"{{!currentQuery.inference ? {{'core.require.inferred' | translate}} : ' '}}"`?

Reply:
No this is not supported